src/numerics.cpp

```diff
@@ -286,6 +286,39 @@ void Integrator1DFourier::compute(const function<double(double)>& func){
 		  qtab, &sol, &err);
 }
 
+
```

Comment:
I am getting some compilation warnings from lines 309 and 310. Please fix them

Reply:
This is related to your 1D integrator. If I remember correctly you said you wanted to work on this.

Reply:
True, I'll close this issue then